dev-games/openscenegraph-openmw/openscenegraph-openmw-3.6_p20210610.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1,2} )
+
+WX_GTK_VER="3.0-gtk3"
+inherit cmake flag-o-matic lua-single wxwidgets
+
+DESCRIPTION="OpenMW-specific fork of OpenSceneGraph"
+HOMEPAGE="https://github.com/OpenMW/osg"
+MY_COMMIT="b02abe200c4847e73b887b064a89ea1758a5b733"
+SRC_URI="https://github.com/OpenMW/osg/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/osg-${MY_COMMIT}"
+
+LICENSE="wxWinLL-3 LGPL-2.1"
+SLOT="0/162" # NOTE: CHECK WHEN BUMPING! Subslot is SOVERSION
+KEYWORDS="~amd64 ~x86"
+IUSE="collada curl dicom debug doc egl examples ffmpeg fltk fox gdal
+gif glut gstreamer jpeg las lua openexr openinventor osgapps pdf png
+sdl sdl2 svg tiff truetype vnc wxwidgets xrandr +zlib"
+
+REQUIRED_USE="dicom? ( zlib )
+	lua? ( ${LUA_REQUIRED_USE} )
+	openexr? ( zlib )
+	sdl2? ( sdl )"
+
+# TODO: FBX, GTA, NVTT, OpenVRML, Performer
+BDEPEND="
+	app-arch/unzip
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )
+"
+RDEPEND="
+	!dev-games/openscenegraph
+	media-libs/mesa[egl(+)?]
+	virtual/glu
+	virtual/opengl
+	x11-libs/libSM
+	x11-libs/libXext
+	collada? ( dev-libs/collada-dom:= )
+	curl? ( net-misc/curl )
+	examples? (
+		fltk? ( x11-libs/fltk:1[opengl] )
+		fox? ( x11-libs/fox:1.6[opengl] )
+		glut? ( media-libs/freeglut )
+		sdl2? ( media-libs/libsdl2 )
+		wxwidgets? ( x11-libs/wxGTK:${WX_GTK_VER}[opengl,X] )
+	)
+	ffmpeg? ( media-video/ffmpeg:0= )
+	gdal? ( sci-libs/gdal:= )
+	gif? ( media-libs/giflib:= )
+	gstreamer? (
+		media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0
+	)
+	jpeg? ( virtual/jpeg:0 )
+	las? ( >=sci-geosciences/liblas-1.8.0 )
+	lua? ( ${LUA_DEPS} )
+	openexr? (
+		media-libs/ilmbase:=
+		media-libs/openexr:=
+	)
+	openinventor? ( media-libs/coin )
+	pdf? ( app-text/poppler[cairo] )
+	png? ( media-libs/libpng:0= )
+	sdl? ( media-libs/libsdl )
+	svg? (
+		gnome-base/librsvg
+		x11-libs/cairo
+	)
+	tiff? ( media-libs/tiff:0 )
+	truetype? ( media-libs/freetype:2 )
+	vnc? ( net-libs/libvncserver )
+	xrandr? ( x11-libs/libXrandr )
+	zlib? ( sys-libs/zlib )
+"
+DEPEND="${RDEPEND}
+	dev-libs/boost
+	x11-base/xorg-proto
+"
+
+PATCHES=(
+	"${FILESDIR}"/openscenegraph-3.6.3-cmake.patch
+	"${FILESDIR}"/openscenegraph-3.6.3-docdir.patch
+	"${FILESDIR}"/openscenegraph-3.6.5-use_boost_asio.patch
+	"${FILESDIR}"/openscenegraph-3.6.5-cmake_lua_version.patch
```

Comment:
Could optionally migrate away from this patch using same method used in openmw-9999, not that it's as important given it's not moving -9999 target.

As chewi mentioned, these patches were done before the -D method was noticed.

Reply:
The patch looks simple enough, and they are using a custom FindLua52 module anyway, instead of FindLua